modin/core/io/__init__.py

```diff
@@ -41,4 +44,5 @@
     "SQLDispatcher",
     "ExcelDispatcher",
     "PickleExperimentalDispatcher",
+    "CustomTextExperimentalDispatcher",
```

Comment:
@anmyachev, why do we put experimental features in the base core instead of experimental one?